### ISSUE TITLE
html2canvas adding unknown spacing in captured image in react-grid-layout

### ISSUE THREAD

**Issue by utkarsh22garg** [niklasvh/html2canvas#2405]
### Bug reports:

Hi Team,
I am using html2canvas to capture the screenshot of the layout created with React-Grid-Layout. But on capturing the page screenshot,  it seems html2canvas is adding some spacing as seen in the picture below. You can see the reproducer link on [Codesandbox](https://codesandbox.io/s/gallant-mayer-6w095). There is a Screenshot button on the bottom of the page. On clicking the button, the screenshot gets appended on the bottom of the page.

#### Original Layout
![image](https://user-images.githubusercontent.com/31355903/97840773-a03a8400-1d0a-11eb-9757-f29cd5386af9.png)

#### Captured Screenshot
![image](https://user-images.githubusercontent.com/31355903/97840816-b8120800-1d0a-11eb-8826-cca00fcd09cd.png)

### Specifications:

 * html2canvas version tested with: 1.0.0-rc.7
 * Browser & version: Chrome - 86
 * Operating system: Windows 10

**Comment by ThanhNguyenDuc2605** [comment]
please awser the question i also have the same issue like him

**Comment by utkarsh22garg** [comment]
Is there any progress on this ?

**Comment by chriswalz-bg** [comment]
Same problem here

**Comment by sachin-sprinkle** [comment]
Hey, Did you guys found any workaround for this? Facing the same problem. Any Idea how to achieve this or any alternative way?

**Comment by utkarsh22garg** [comment]
@niklasvh Can you please take a look at it?

**Comment by voduchau** [comment]
+1

**Comment by voduchau** [comment]
add `useCSSTransforms={false}` to prop layout
it work for me

**Comment by utkarsh22garg** [comment]
> add `useCSSTransforms={false}` to prop layout
> it work for me

Yeah this works. But disabling css transforms degrades the painting performance of react-grid-layout as per their [docs](https://chrome.google.com/webstore/detail/incognito/kencjkgapindpgehbgolojoocgpcepfk). So I still need css transforms support in react-grid-layout.
